parsers/s01-parse/crowdsecurity/postfix-logs.yaml

```diff
@@ -58,4 +58,38 @@ statics:
       expression: "evt.Parsed.remote_host"
     - meta: log_type
       value: postfix
-
+---
+onsuccess: next_stage
+filter: "evt.Parsed.program == 'postfix/cleanup'"
+name: krvtz/postfix-spam
+description: "Parse postfix logs"
+nodes:
+  - grok:
+      apply_on: message
+      pattern: 'milter-reject: END-OF-MESSAGE from %{POSTFIX_HOSTNAME:remote_host}\[%{IP:remote_addr}\]: 4.7.1 Spam message rejected'
+      statics:
+        - meta: log_type_enh
+          value: spam-attempt
+  - grok:
+      apply_on: message
+      pattern: ' %{POSTFIX_HOSTNAME:remote_host}\[%{IP:remote_addr}\]: 450 4.1.1 <[a-zA-Z0-9@-]+>: Recipient address rejected'
+      statics:
+        - meta: log_type_enh
+          value: spam-attempt
+  - grok:
+      apply_on: message
+      pattern: ' %{POSTFIX_HOSTNAME:remote_host}\[%{IP:remote_addr}\]: SASL PLAIN authentication failed'
```

Comment:
Wouldn't this already be covered by the

`'warning: %{POSTFIX_HOSTNAME:remote_host}\[%{IP:remote_addr}\]: SASL ((?i)LOGIN|PLAIN|(?:CRAM|DIGEST)-MD5) authentication failed:%{GREEDYDATA:message_failure}'`

but I guess the application doesnt match

Reply:
You are actually right - it was my mistake to miss the `SASL` parser and then include mine in `cleanup` block, where it would be never matched. Now fixing this in another commit.  So we're just left with the spam rules.